terrautils/extractors.py

```diff
@@ -18,10 +18,94 @@
 from osgeo import gdal, osr
 from PIL import Image
 
+from pyclowder.extractors import Extractor
 from pyclowder.collections import get_datasets
 from pyclowder.datasets import get_file_list, submit_extraction as submit_ext_ds
 from pyclowder.files import submit_extraction as submit_ext_file
 
+CLOWDER_SPACE = "58da6b924f0c430e2baa823f"
+
+
+def add_arguments(parser):
+
+    # TODO: shouldn't this be part of pyclowder?
```

Comment:
only extractors that would create nested hierarchies of datasets would need this, because it defines root to build those under. most other terra extractors (up to now) and every other project extractor just adds extracted results to existing dataset so this is not a widely needed flag. I think it's fine in terrautils instead of just that extractor however, as we might need it for the data release processing to separate raw_Data datasets from Level_1+.